packages/block-editor/src/hooks/typography.js

```diff
@@ -30,6 +30,7 @@ function omit( object, keys ) {
 const LETTER_SPACING_SUPPORT_KEY = 'typography.__experimentalLetterSpacing';
 const TEXT_TRANSFORM_SUPPORT_KEY = 'typography.__experimentalTextTransform';
 const TEXT_DECORATION_SUPPORT_KEY = 'typography.__experimentalTextDecoration';
+const TEXT_SHADOW_SUPPORT_KEY = 'typography.__experimentalTextShadow';
```

Comment:
In recent times, we've avoided landing more `__experimental` block supports. Once they are in a release they tend to get used and we have to provide backward compatibility anyway. It's easiest to just land new supports as stable.
```suggestion
const TEXT_SHADOW_SUPPORT_KEY = 'typography.textShadow';
```